apps/hash-frontend/src/components/hooks/use-users.ts

```diff
@@ -37,9 +37,9 @@ export const useUsers = (
       constrainsPropertiesOn: { outgoing: 0 },
       constrainsLinksOn: { outgoing: 0 },
       constrainsLinkDestinationsOn: { outgoing: 0 },
-      isOfType: { outgoing: 1 },
-      hasLeftEntity: { incoming: 1, outgoing: 1 },
-      hasRightEntity: { incoming: 1, outgoing: 1 },
```

Comment:
Just to check if this is the correct way: For a given entity this will traverse the ***outgoing*** link. So in the following example, it will also discover `B` when querying for `A`:
```
Entity A --Link-> Entity B
```

Reply:
Yes, that's correct. For users we want to retrieve orgs they are a member of. This is modelled as follows:

```
User -> OrgMembership -> Org
```